### PR TITLE
Added ktlint to project modules and created gradle synchronization

### PR DESCRIPTION
**Pull request by Faka-GLB**
Integrated ktlint to all four project modules (app, data, di and domain).

Moved all dependencies to dependencies.gradle in the buildsync directory.

Also deleted example tests.

Project tree:
Rick and morty modules

### CODE REVIEW

**Review by mauroLonghi — APPROVED** [review]

**Review by BQuinteros — APPROVED** [review]

File: data/build.gradle [hunk 1]
```diff
@@ -31,14 +31,24 @@ android {
...
+    implementation lib.appCompat
+
+    //UI
+    implementation lib.material
+    implementation lib.constraintLayout
+
+    //test
+    testImplementation lib.junit
+    androidTestImplementation lib.extJunit
+    androidTestImplementation lib.espresso
```
Comment (nicolbidone) [inline data/build.gradle]: are you sure you need these dependencies here?

File: di/build.gradle [hunk 2]
```diff
@@ -31,15 +31,25 @@ android {
...
+    implementation lib.appCompat
+
+    //UI
+    implementation lib.material
+    implementation lib.constraintLayout
+
+    //test
+    testImplementation lib.junit
+    androidTestImplementation lib.extJunit
+    androidTestImplementation lib.espresso
```
Comment (nicolbidone) [inline di/build.gradle]: are you sure you need these dependencies here?

File: data/build.gradle (hunk 1 above)
Comment (Faka-GLB) [reply data/build.gradle]: Removed test dependencies

File: di/build.gradle (hunk 2 above)
Comment (Faka-GLB) [reply di/build.gradle]: Removed test dependencies

**Review by nicolbidone — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Pull request merged**